common/templates/inputs/rangeInputs.html

```diff
@@ -9,22 +9,22 @@
                     <!-- int2 -->
                     <div ng-switch-when="int2">
                         <div class="chaise-input-control has-feedback">
-                            <input type="number" class="range-min" min="{{intMin}}" max="{{intMax}}" placeholder="{{absMin}}" ng-model="model.min" integer disable-arrows>
+                            <input type="text" class="range-min" chaise-min="{{intMin}}" chaise-max="{{intMax}}" placeholder="{{absMin}}" ng-model="model.min" integer disable-arrows>
                             <chaise-clear-input btn-class="min-clear" click-callback="model.min=''" show="model.min || (numericMin.$invalid && numericMin.$dirty)"></chaise-clear-input>
                         </div>
                     </div>
                     <!-- int4 -->
                     <div ng-switch-when="int4">
                         <div class="chaise-input-control has-feedback">
-                            <input type="number" class="range-min" min="{{intMin}}" max="{{intMax}}" placeholder="{{absMin}}" ng-model="model.min" integer disable-arrows>
+                            <input type="text" class="range-min" chaise-min="{{intMin}}" chaise-max="{{intMax}}" placeholder="{{absMin}}" ng-model="model.min" integer disable-arrows>
```

Comment:
I think you can also remove the `disable-arrows` directive from here. It seems like it's meant to " disables the actions that HTML attaches to those input types for up/down", so it's not needed anymore.